partner-onboarder/copy_cm.sh

```diff
@@ -5,6 +5,6 @@
 COPY_UTIL=./copy_cm_func.sh
 DST_NS=injiweb
 
-$COPY_UTIL configmap global default $DST_NS
-$COPY_UTIL configmap keycloak-env-vars keycloak $DST_NS
-$COPY_UTIL configmap keycloak-host keycloak $DST_NS
+$COPY_UTIL configmap inji-stack-config default $DST_NS
+$COPY_UTIL configmap keycloak-env-vars keycloak $DST_NS || { echo "Ignore the error if Keycloak values have been added to values.yaml."; true; }
+$COPY_UTIL configmap keycloak-host keycloak $DST_NS || { echo "Ignore the error if Keycloak values have been added to values.yaml."; true; }
```

Comment:
echo "Ignore the error if Keycloak is deployed externally and details of the same have been added to values.yaml."; true;